platform/machine/aws/cluster.go

```diff
@@ -84,7 +84,7 @@ func (ac *cluster) NewMachine(userdata string) (platform.Machine, error) {
 
 	conf.CopyKeys(keys)
 
-	instances, err := ac.api.CreateInstances(ac.Name(), ac.Name(), conf.String(), 1, true)
+	instances, err := ac.api.CreateInstances(ac.Name(), ac.Name(), conf.String(), 1)
```

Comment:
Yes, I realized that after the fact.  Changing it to `CreateInstance` would simplify the driver, or alternatively `platform` could learn to request multiple machines at once.  Currently `platform` handles multiple launches by spawning a goroutine for each one, so I think the only thing we're losing with the current system is API throttling headroom -- which means your implied cleanup could make sense.  Thoughts?